ansible/playbook_rollout.yml

```
@@ -49,3 +49,5 @@
     - role: post_deployment
     - role: moin-schule-sync
       when: WITH_MOIN_SCHULE is defined and WITH_MOIN_SCHULE|bool
+    - role: common-cartridge
+      when: WITH_COMMON_CARTRIDGE is defined and WITH_COMMON_CARTRIDGE|bool
```

It's not necessary to check the when condition as is also at add as conditions for the job in the ansible role
```suggestion
    - role: common-cartridge
```
And put it after the `schulcloud-client-core` role and bevore the `post_deployment` role. 
The `post_deployment` ansible role shoud be the last role, it marks that the deployment is finished

---

Done